inst/scripts/insert_image.R

```diff
@@ -0,0 +1,59 @@
+local({
+  txt_input = function(..., width = '100%') shiny::textInput(..., width = width)
+
+  blogfile = rstudioapi::getActiveDocumentContext()
+  if(blogfile$path == '') stop('Please select the blog post source file before using this addin', call. = FALSE)
+
+  imgdir = file.path(
+    gsub('/content/.*', '', normalizePath(blogfile$path)),
+    'static',
+    dirname(gsub('.*content/', '', normalizePath(blogfile$path))),
```

Comment:
`normalizePath(blogfile$path)` appeared twice -- probably a sign of factoring it out

Reply:
I now save this in a variable, instead of running the function multiple times